src/App.js

```diff
@@ -2,13 +2,16 @@ import React from 'react';
 
 import { CreateGlobalStyle } from './fonts/global-fonts';
 import { Home } from './pages/home';
+import { AvaliacaoProfs } from './pages/avaliacao-profs'
+import { MaterialApoio } from './pages/material-apoio'
+import { MinhasDisciplinas } from './pages/minhas-disciplinas'
 
 function App() {
   return (
     <>
       <CreateGlobalStyle/>
-      <Home />
```

Comment:
O ideal é que o Home fique aqui, vc só altera para ver como ta ficando. Mas quando for mergear na branch principal deixa assim por enquanto